checkov/main.py

```diff
@@ -415,6 +425,40 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
     return None
 
 
+def report_contributor_metrics(repository: str) -> None:  # ignore: type
+
+    def _parse_gitlog() -> dict[str, Any] | None:
+        process = subprocess.Popen(['git', 'shortlog', '-ne', '--all', '--since', '"90 days ago"', '--pretty="commit-%ct"', '--reverse'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)  # nosec
+        out, err = process.communicate()
+        if err:
+            logger.info('Failed to collect contributor metrics')
+            return None
+        # split per contributor
+        list_of_contributors = out.decode('utf-8').split('\n\n')
+        return {"repository": repository,
+                "contributors": list(map(lambda contributor: _process_contributor(contributor),
+                                         list(filter(lambda x: x, list_of_contributors))
+                                         ))
+                }
+
+    def _process_contributor(contributor: str) -> str:
+        splittedList = contributor.split('\n')
+        user = splittedList[0]
+        commit = splittedList[1]
+        return user[0:user.find('(')] + commit[commit.find('-') + 1:len(commit)].strip('\"')
```

Comment:
The `0` and `len(commit)` can be omitted

```suggestion
        return user[:user.find('(')] + commit[commit.find('-') + 1:].strip('\"')
```